_build_cfg.yml

```diff
@@ -26,11 +26,11 @@ Dir: v2_changes
 Topics:
   - Name: Overview
     File: overview
-  - Name: Cartridges vs Images
-    File: carts_vs_images
   - Name: Applications
     File: applications
-  - Name: Terminology
+  - Name: Cartridges vs Images
+    File: carts_vs_images
+  - Name: Terminology Changes
     File: terminology
 
 ---
```

Comment:
These will most likely be renamed and restructured once the content is in place and finalized.